### DISCUSSION TITLE
Hiding secure variables

### DISCUSSION THREAD

**Discussion by ShawInnes** [mobile-dev-inc/Maestro#1110] [Q&A] [answered]
What a great project! 

I've just been automating some login flows and wondered whether there's a way to stop the output from displaying secure variables?

For example, it shows the variables here but when it runs the steps it will replace the ${EMAIL} and ${PASSWORD} with the values.  

![image](https://github.com/mobile-dev-inc/maestro/assets/610688/5e880aa5-d10a-431b-a48d-bd821fcf5eca)

It'd be great if there was a way to mark a step as secure/sensitive and not show the value in the log. I did search and look around the documentation without any success, so I apologise in advance if I just missed the instructions!

**Comment by axelniklasson** [comment, marked as answer]
Hi there @ShawInnes! That is indeed not something that is supported today but feel free to raise a feature request [here](https://github.com/mobile-dev-inc/maestro/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=%5BFeature+Request%5D). Thanks!
